web/src/main/webapp/dataViews/tableView.ts

```diff
@@ -519,7 +519,8 @@ export class TableView extends RemoteTableObjectView implements IScrollTarget {
                 this.contextMenu.addItem({text: "Heavy hitters...", action: () => this.heavyHitters(false) });
                 //this.contextMenu.addItem({text: "Heavy hitters (Streaming)...", action: () => this.heavyHitters(true) });
                 //this.contextMenu.addItem({text: "Select numeric columns", action: () => this.selectNumericColumns()});
-                this.contextMenu.addItem({text: "PCA...", action: () => this.pca() });
+                this.contextMenu.addItem({text: "PCA", action: () => this.pca(true) });
```

Comment:
The ellipses after "PCA" are important, they indicate that another menu will be shown after this one. Please put them back.